automatic/videoder/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,20 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsPath   = Split-Path $MyInvocation.MyCommand.Definition
+
+$packageArgs = @{
+  packageName    = 'videoder'
+  fileType       = 'exe'
+  file           = Get-Item "$toolsPath\*.exe"
+  file64         = Get-Item "$toolsPath\*.exe"
```

Comment:
Is this section correct? What do you think @majkinetor?

Reply:
If the same file i used for both, you don't need file64, just file. But that will work too.

Reply:
There are two versions of this binary, though their names are the same.